### DISCUSSION TITLE
Should Evergreen support all release/channels for a specific product?

### DISCUSSION THREAD

**Discussion by aaronparker** [EUCPilots/evergreen-module#138] [General]
💡 Should Evergreen support all release/channels, such as beta, insider, dev, etc. for a specific product and report on whatever is available? Think Edge, Chrome, Visual Studio Code etc., that provide access to these types of releases in addition to the current release.

**Comment by Deyda** [comment]
Yes this would be great.

**Comment by aaronparker** [comment]
Along these lines - should we be adding ARM64 support were it exists?
